src/multisketch.rs

```diff
@@ -196,6 +191,67 @@ impl MultiSketch {
 
         self
     }
+    pub fn remove_metadata(
+        &mut self,
+        output_file_name: &str,
+        genome_ids_to_remove: &[String],
+    ) -> std::io::Result<()> {
+        println!("{}", self);
+        let mut new_sketch_metadata: Vec<Sketch> = Vec::with_capacity(self.sketch_metadata.len());
```

Comment:
Wrong capacity, it should be `self.sketch_metadata.len() - genome_ids_to_remove.len()`